### PR TITLE
Add cross-project links

### PR DESCRIPTION
**Pull request by MikeWasson**
@danroth27 These refs look right, but I'm getting "undefined label" when I build locally. Can you take a look?

### CODE REVIEW

**Comment by danroth27** [comment]
I think you need to build and update aspnet.inv in the MVC project.

**Comment by MikeWasson** [comment]
Ah, got it. I had to build aspnet, then copy aspnet/objects.inv over to mvc/aspnet.inv.

Intuitively I would expect aspnet.inv to be a build artifact and not checked into the repo?

**Comment by danroth27** [comment]
It is a build artifact of the aspnet project :smile:.

If you don't add this file to the MVC project then MVC will try to fetch it over the network, which can cause build flakiness.

File: mvc/tutorials/mvc-with-entity-framework.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ In this article:
     - `Add a delete page`_
     - `Wrapping up`_
 
-You can browse the source code for the sample app on `GitHub <https://github.com/aspnet/Docs/tree/master/docs/mvc/tutorials/mvc-with-entity-framework/sample>`_.
+You can browse the source code for the sample app on `GitHub <https://github.com/aspnet/Docs/tree/master/mvc/tutorials/mvc-with-entity-framework/sample>`__.
```
Comment (danroth27) [inline mvc/tutorials/mvc-with-entity-framework.rst]: Why the additional underscore?

**Comment by danroth27** [comment]
:shipit:

Also, please note that the DNX docs will be moving over to http://dotnet.readthedocs.org soon.

File: mvc/tutorials/mvc-with-entity-framework.rst (hunk 1 above)
Comment (MikeWasson) [inline mvc/tutorials/mvc-with-entity-framework.rst]: http://stackoverflow.com/a/14067756/1261807

I guess the other option is a ref but this seemed simpler.

**Comment by MikeWasson** [comment]
cae7c02bb8dc8be5be52b021c676fb53eb7133b5

**Pull request closed without merging**